docs/releases/5.1.txt

```diff
@@ -234,6 +234,9 @@ Management Commands
   each operation to highlight :class:`operation categories
   <django.db.migrations.operations.base.OperationCategory>`.
 
+* Introduce a :djadmin:`listurls` command that lists the URLs in the application,
```

Comment:
5.1 is now in alpha so new features added now will be released in 5.2. Therefore please could you move this to `5.2.txt`?